homeassistant/components/sensor/serial_pm.py

```diff
@@ -21,7 +21,6 @@
 CONF_BRAND = "brand"
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
-    vol.Required(CONF_PLATFORM): 'serial_pm',
```

Comment:
You can try adding a `print(p_validated)` before this line; https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/bootstrap.py#L135